erts/emulator/beam/jit/arm/generators.tab

```diff
@@ -571,6 +571,50 @@ gen.bs_match(Fail, Ctx, N, List) {
     int fixed_args;
     int i;
 
+    /*
+     * If a BEAM file produced by a later version of Erlang/OTP
+     * is accidentally loaded into an earlier version, ensure
+     * that the loading fails (as opposed to crashing the runtime)
+     * if there are any unknown sub commands.
+     */
+    i = 0;
```

Comment:
Can we break this part out into a common macro in the top-level `generators.tab`? It seems a bit repetitive.

Reply:
Yes, it is repetitive, but it is only used by the two JITs, not the emulator.